modules/basis/basis-datasource/samples/kube.yml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dynamic-db
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dynamic-db
+  template:
+    metadata:
+      labels:
+        app: dynamic-db
+    spec:
+      containers:
+        - name: dynamic-db
+          image: radiatorspring/dynamic-db:latest
+          env:
+            - name: DefaultDB_url
+              value: jdbc:sap://43dcda2a-75aa-459f-928f-b92ca222c506.hana.trial-eu10.hanacloud.ondemand.com:443/db
+            - name: DefaultDB_username
+              value: DBADMIN
+            - name: DefaultDB_password
```

Comment:
How about `DEFAULT_DB_PASSWORD` instead of `DefaultDB_password`?

Reply:
same